src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Bindings.JavaDependencyVerification.targets

```diff
@@ -0,0 +1,38 @@
+<!--
+***********************************************************************************************
+Xamarin.Android.Bindings.JavaDependencyVerification.targets
+
+This file contains MSBuild targets used to enable features using POM files to ensure
+that Java binding dependencies are satisfied.
+
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="Xamarin.Android.Tasks.GetMicrosoftNuGetPackagesMap" AssemblyFile="Xamarin.Android.Build.Tasks.dll" />
+  <UsingTask TaskName="Xamarin.Android.Tasks.JavaDependencyVerification" AssemblyFile="Xamarin.Android.Build.Tasks.dll" />
+
+  <Target Name="_VerifyJavaDependencies"
+      Condition=" '@(AndroidLibrary->Count())' != '0' "
```

Comment:
Even though we "strongly suggest" people use the new item name, there are some using the old one.

Can this check `$(_AndroidGenerateManagedBindings)` instead?

https://github.com/xamarin/xamarin-android/blob/b812506b9c9dab88a5bc48f561efc7290309a401/src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Bindings.Core.targets#L47-L53

Reply:
As written, none of these new features will function on the "legacy" item names.

For example:

```
<!-- Satisfies the Java dependency verifier -->
<AndroidLibrary Include="mylib.jar" Pack="false" JavaArtifact="com.example.mylib" JavaVersion="1.0.0" />

<!-- Does not satisfy the Java dependency verifier -->
<InputJar Include="mylib.jar" JavaArtifact="com.example.mylib" JavaVersion="1.0.0" />
```

Running it on a project that does not have `<AndroidLibrary>` will waste time.

Maybe we should take a stronger stance and add a deprecation warning for the legacy items in .NET 9 and remove them in .NET 10?

Reply:
We can *begin* taking a stance and start warning on usage of legacy item names.  That doesn't require that we remove support for them entirely, but it's required *before* we remove support for them entirely…